src/dashboard/src/pages/AllowedIP/index.tsx

```diff
@@ -128,7 +145,10 @@ const UpdateForm: FunctionComponent<UpdateFormProps> = ({ onSubmit }) => {
 
   return (
     <Card component="form" onSubmit={handleSubmit(onSubmit)}>
-      <CardHeader title="Update Allowed IP"/>
+      <CardHeader
+        title="Update Allowed IP"
+        subheader="For Non-Lab clusters. It would take up to 5 minutes to make it available."
```

Comment:
It takes several minutes to take effect for Azure clusters. Non-Azure clusters do not have IP allow list.